test/functional/user_specified_tag_tests.py

```diff
@@ -0,0 +1,87 @@
+import os
+import tempfile
+import unittest
+
+from tito.cli import CLI
+from tito.common import run_command
+from tito.compat import getoutput
+
+TEST_SPEC = """
+Name:           hello_tito
+Version:        0.1.7
+Release:        1%{?dist}
+Summary:        testing package
+
+License:        Public Domain
+Source0:        hello_tito-%{version}.tar.gz
+
+%description
+
+
+%prep
+%autosetup
+
+
+%build
+
+%install
+
+%files
+
+%changelog
+"""
+
+
+def tito(argstring):
+    """ Run Tito from source with given arguments. """
+    return CLI().main(argstring.split(' '))
+
+
+class UserSpecifiedTagTest(unittest.TestCase):
```

Comment:
I would request a rename, user specified doesn't seem right here to me as it's not entirely clear what that means, users can always specify which tagger to use. This is more about custom tag formats so I would suggest CustomFormatTagger or FormatTagger, open to other suggestions as well.